README.md

```diff
@@ -53,6 +55,11 @@ use fastembed::{TextEmbedding, InitOptions, EmbeddingModel};
 // With default InitOptions
 let model = TextEmbedding::try_new(Default::default())?;
 
+// Alternatively, here a "bring your own" model could be loaded
+// Users will need to manually pass in the bytes of the relevant model files
+// This includes  the .onnx file itself and json files to constitute the TokenizerFiles struct)
+let custom_model = TextEmbedding::try_new_from_user_defined(UserDefinedEmbeddingModel, options);
+
```

Comment:
This should be a separate snippet probably.
Since it's not straightforward how a user can instantiate `UserDefinedEmbeddingModel` and that `read_file_to_bytes()` exists.

Reply:
Sorry, what do you mean by separate snippet? 

I'll add some visibility for the read_file_to_bytes function. (Just FYI, I wasn't intending for that to be super exposed because it's not really the job of this library, but agree it would be better DX.)

Reply:
Like a separate section in the README.md with a code example to load custom models.

Examples are the best docs, I believe.

Reply:
Understood - thanks. I'll submit over the next few days.